rocketpy/control/controller.py

```diff
@@ -2,42 +2,58 @@
 
 
 class Controller:
-    """A class for storing and running controllers on a rocket."""
+    """A class for storing and running controllers on a rocket. Controllers
+    have a controller function that is called at every step of the simulation
+    at a specified sampling rate. The controller function can access and modify
+    the objects that are passed to it. The controller function also stores the
+    variables of interest in the objects that are passed to it."""
 
     def __init__(
         self,
-        observed_objects,
+        interactable_objects,
         controller_function,
         sampling_rate,
+        initial_observed_variables=None,
         name="Controller",
     ):
         """Initialize the class with the controller function and the objects to
         be observed.
 
         Parameters
         ----------
-        observed_objects : list
-            A list of objects to be observed by the controller. It can be any
-            python object. This list will be passed to the controller function
-            as positional arguments, meaning that the order of the objects in
-            this list matters. These objects will then be able to be accessed
-            and modified by the controller function.
+        interactable_objects : list
+            A collection of objects that the controller function can access and
+            potentially modify. This list is flexible and can include any Python
```

Comment:
"This list is flexible and can include any Python object"

this sentence let me a bit lost here. What would be examples of "any Python object"? What do you mean by that?

Reply:
Well, the user can pass any type of object in this list. If they want to use an object of a class that they defined they can.

Since these are passed directly to the controller_function, and the controller_function is completely defined by the user, he can pass whatever he wants in here.

I will try to improve the docs a bit here